packages/gatsby-theme-docs/src/modules/sso/hooks/use-authentication.ts

```diff
@@ -2,10 +2,37 @@ import { useContext, useEffect, useState } from 'react';
 import useSWR from 'swr';
 import { useAuth0 } from '@auth0/auth0-react';
 import { useAuthToken } from '../../self-learning/hooks/use-auth-token';
-import ConfigContext from '../../../components/config-context';
+import ConfigContext, {
+  EFeatureFlag,
+  isFeatureEnabled,
+} from '../../../components/config-context';
 import { fetcherWithToken } from '../../self-learning/hooks/hooks.utils';
 import { getCookieValue } from '../utils/common.utils';
 
+export const LOCAL_STORAGE_SESSION = 'user_session';
```

Comment:
Ok for renaming the the item. 

About the content itself, I'm not sure to understand what you mean. 
I think that the content of this localStorage item is not that important, in fact, I believe we could not store the userid and simply add a fixed meaningless string (it could be `session`). But I see no value in adding the JSON serializing/deserializing overhead. If, at some point we're going to store more session information we could think about it although the string access to the localStorage is so handy that even a localStorage item for each property doesn't sound so bad.

Reply:
I renamed the item to `user_auth0_session`, I think now it's more explicit.
I also had another thought about removing the userid from the content. I think it's better to have it as we can cover the corner case where [a session exist but from a different user](https://github.com/commercetools/commercetools-docs-kit/pull/1756/files#diff-23fceb793741100d858cdbdb91f19026e31867ae49ee5d56347c666c821c71aaR17-R25)
Also, if we leave the user id in the content we can reduce the number of local storage set events to the minimum (only when there's no session or the session belong to a different user)